cmd/mapt/cmd/aws/constants/constants.go

```diff
@@ -21,6 +21,20 @@ const (
 	MACFixedLocationDesc string = "if this flag is set the host will be created only on the region set by the AWS Env (AWS_DEFAULT_REGION)"
 	MACDHID              string = "dedicated-host-id"
 	MACDHIDDesc          string = "id for the dedicated host"
+
+	ParamLocation                = "location"
+	ParamLocationDesc            = "location for created resources in case spot flag (if available) is not passed"
+	DefaultLocation              = "West US"
```

Comment:
That one should no be needed this is coming from Env: DEFAULT_AWS_REGION

Reply:
Ok, removed 👍